src/duckling/util/keyboard.service.ts

```diff
@@ -32,3 +32,29 @@ export class KeyboardService implements OnDestroy {
         this._heldKeys[event.keyCode] = false;
     }
 }
+
+
+/**
+ * Lookup functions for keyboard key codes
+ */
+export class KeyboardCodeLibrary {
```

Comment:
I think this should be a constant object of constants instead of a class of static functions.